src/FSharp.Control.TaskSeq/TaskSeq.fsi

```diff
@@ -154,414 +175,872 @@ module TaskSeq =
+    /// This function is non-blocking while it builds the list.
+    /// </summary>
+    ///
+    /// <param name="source">The input task sequence.</param>
+    /// <returns>The resulting list.</returns>
+    /// <exception cref="T:ArgumentNullException">Thrown when the input sequence is null.</exception>
     val toListAsync: source: taskSeq<'T> -> Task<'T list>
 
-    /// Unwraps the taskSeq as a Task<ResizeArray<_>>. This function is non-blocking.
+    /// <summary>
+    /// Builds a resizable array asynchronously from the input task sequence in <paramref name="source" />.
+    /// This function is non-blocking while it builds the resizable array.
+    /// </summary>
+    ///
+    /// <param name="source">The input task sequence.</param>
+    /// <returns>The resulting resizable array.</returns>
+    /// <exception cref="T:ArgumentNullException">Thrown when the input sequence is null.</exception>
     val toResizeArrayAsync: source: taskSeq<'T> -> Task<ResizeArray<'T>>
 
-    /// Unwraps the taskSeq as a Task<IList<_>>. This function is non-blocking.
+    /// <summary>
+    /// Builds an <see cref="IList&lt;'T>" /> asynchronously from the input task sequence in <paramref name="source" />.
+    /// This function is non-blocking while it builds the IList.
+    /// </summary>
+    ///
+    /// <param name="source">The input task sequence.</param>
+    /// <returns>The resulting IList interface.</returns>
+    /// <exception cref="T:ArgumentNullException">Thrown when the input sequence is null.</exception>
     val toIListAsync: source: taskSeq<'T> -> Task<IList<'T>>
 
-    /// Create a taskSeq of an array.
+    /// <summary>
+    /// Views the given <see cref="array" /> as a task sequence, that is, as an <see cref="IAsyncEnumerable&lt;'T>" />.
+    /// </summary>
+    ///
+    /// <param name="source">The input array.</param>
+    /// <returns>The resulting task sequence.</returns>
+    /// <exception cref="T:ArgumentNullException">Thrown when the input array is null.</exception>
     val ofArray: source: 'T[] -> taskSeq<'T>
 
-    /// Create a taskSeq of a list.
+    /// <summary>
+    /// Views the given <see cref="list" /> as a task sequence, that is, as an <see cref="IAsyncEnumerable&lt;'T>" />.
+    /// </summary>
+    ///
+    /// <param name="source">The input list.</param>
+    /// <returns>The resulting task sequence.</returns>
     val ofList: source: 'T list -> taskSeq<'T>
 
-    /// Create a taskSeq of a seq.
+    /// <summary>
+    /// Views the given <see cref="seq" /> as a task sequence, that is, as an <see cref="IAsyncEnumerable&lt;'T>" />.
+    /// </summary>
+    ///
+    /// <param name="source">The input sequence.</param>
+    /// <returns>The resulting task sequence.</returns>
+    /// <exception cref="T:ArgumentNullException">Thrown when the input sequence is null.</exception>
     val ofSeq: source: seq<'T> -> taskSeq<'T>
 
-    /// Create a taskSeq of a ResizeArray, aka List.
+    /// <summary>
```

Comment:
can I mutate the resizearray after I apply this operator? does it take a defensive copy? why use this over ofSeq or ofArray? (if its only for symmetry with toResizeArray and thats a bit of a special case/wart, maybe don't provide it?)

Reply:
and there is no ofTaskResizeArray ;)

Reply:
>  why use this over ofSeq or ofArray? (if its only for symmetry with toResizeArray and thats a bit of a special case/wart, maybe don't provide it?)

Yeah, I'm not a big fan of this one. I should probably just drop it. It has been in the library for a while, but people can just use `ofSeq/toSeq` and variants.

Reply:
Btw, the documentation changes came into this PR because I accidentally based it off of #136. I will put your comments in there.

And yes, it should make a defensive copy.

Reply:
This one is definitely up for being dropped.